instrumentation/koala/lib/opentelemetry/instrumentation/koala/patches/instrumentation.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Instrumentation
+    module Koala
+      module Patches
+        # Module to prepend to Koala::Facebook::API for instrumentation
+        module Api
+          VERBS_TO_SPAN_NAMES = Hash.new { |h, k| h[k] = "Koala #{k}" }
+
+          def graph_call(path, args = {}, verb = 'get', options = {}, &post_processing)
+            tracer.in_span(
+              VERBS_TO_SPAN_NAMES[verb],
+              attributes: {
+                'koala.verb' => verb,
+                'koala.path' => path
+              },
+              kind: :client
+            ) do |_|
+              OpenTelemetry::Common::HTTP::ClientContext.with_attributes('peer.service' => 'facebook') do
+                super
+              end
+            end
```

Comment:
I don't think we want to wrap this in a span - certainly not a client span. I think this should be:
```suggestion
            OpenTelemetry::Common::HTTP::ClientContext.with_attributes('peer.service' => 'facebook', 'koala.verb' => verb, 'koala.path' => path) do
              super
            end
```